### PR TITLE
docs(copy_edits): FRMW-263-Copy-Edits-Pagination

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-263 - Table Footer Pagination via rxPaginate

- [x] Dev LGTM
- [x] Dev LGTM
- [x] Design LGTM

**Copy Edits** for Tables Styleguide

### CODE REVIEW

**Comment by 100stacks** [comment]
screen shot 2015-10-08 at 4 08 30 pm

**Comment by bry4n** [comment]
LGTM

File: demo/styleguide/tables.html [hunk 1]
```diff
@@ -30,8 +30,25 @@ <h3 class="title">Table Footer Pagination via rxPaginate</h3>
...
-            <li><span class="msg-warn">HEADS UP:</span> It's a known issue that the footer wraps into a second line if the table is 50% of the screen. In cases like those, we've used the <code>display:none</code> CSS property for elements to hide all elements not directly related to pagination. See the <a href="https://github.com/rackerlabs/supportservice-ui/blob/master/app/styles/table.less#L40-L50">LESS of narrow tables used in Support Services</a> for an example of this. Note that this has not been implemented in EncoreUI yet.</li>
+            <li>It's a known issue that the footer wraps into a second line if the table is 50% of the screen.  Use <code>display:none</code> CSS property to hide all elements not directly related to pagination.  The LESS for narrow tables used in Support Services for an example of this.  This has not been implemented in EncoreUI.</li>
         </ul>
+        
+    <pre>
+    /* The new pagination style will make small-width columns/
```
Comment (CITguy) [inline demo/styleguide/tables.html]: Put "columns/modals" together on the same line. With it broken between two lines, it looks strange.

File: demo/styleguide/tables.html [hunk 2]
```diff
@@ -30,8 +30,25 @@ <h3 class="title">Table Footer Pagination via rxPaginate</h3>
...
+     *  modals tables look weird. Hide as much elements as possible. 
+     */
+
```
Comment (CITguy) [inline demo/styleguide/tables.html]: Remove empty line.

File: demo/styleguide/tables.html [hunk 3]
```diff
@@ -30,8 +30,25 @@ <h3 class="title">Table Footer Pagination via rxPaginate</h3>
...
+    .rackers-table.teams .pagination,
+    .account-list-table .pagination,
+    .modify-racker-accounts .pagination,
+    .modify-racker-teams .pagination,
+    .modify-team-users .pagination,
+    .modify-team-accounts .pagination {
```
Comment (CITguy) [inline demo/styleguide/tables.html]: I'd simplify these selectors down to a generic one like `.my-custom-class-name .pagination { ... }`

File: demo/styleguide/tables.html [hunk 4]
```diff
@@ -30,8 +30,25 @@ <h3 class="title">Table Footer Pagination via rxPaginate</h3>
...
             <li><span class="msg-info">ENCORE-SPECIFIC:</span> The pagination is visible in this example to show how mark-up will be implemented. Generally, the pagination directive does not display if there is only one page of data available. We use angular.js to make sure this doesn't display.</li>
-            <li><span class="msg-warn">HEADS UP:</span> It's a known issue that the footer wraps into a second line if the table is 50% of the screen. In cases like those, we've used the <code>display:none</code> CSS property for elements to hide all elements not directly related to pagination. See the <a href="https://github.com/rackerlabs/supportservice-ui/blob/master/app/styles/table.less#L40-L50">LESS of narrow tables used in Support Services</a> for an example of this. Note that this has not been implemented in EncoreUI yet.</li>
+            <li>It's a known issue that the footer wraps into a second line if the table is 50% of the screen.  Use <code>display:none</code> CSS property to hide all elements not directly related to pagination.  The LESS for narrow tables used in Support Services for an example of this.  This has not been implemented in EncoreUI.</li>
```
Comment (CITguy) [inline demo/styleguide/tables.html]: Sentence doesn't make sense

````
The LESS for narrow tables used in Support Services for an example of this.
```

Suggest updating to:

```
See the CSS example below for an example.
```

**Comment by 100stacks** [comment]
screen shot 2015-10-09 at 10 26 59 am

**Comment by 100stacks** [comment]
@CITguy comments incorporated :+1:

**Comment by CITguy** [comment]
Oops, I meant to keep the internals for the CSS block. Please put back the content of the CSS block.

```
.my-custom-class-name .pagination {
    & > li.nth-child(1) { display: none; }
    & > li.nth-child(2) { display: none; }
    .pagination-per-page { display: none; }
}
```

**Comment by 100stacks** [comment]
screen shot 2015-10-09 at 11 59 50 am

**Comment by 100stacks** [comment]
@CITguy okay CSS block reverted :+1:

File: demo/styleguide/tables.html [hunk 5]
```diff
@@ -29,9 +29,17 @@ <h3 class="title">Table Footer Pagination via rxPaginate</h3>
...
-            <li><span class="msg-warn">HEADS UP:</span> It's a known issue that the footer wraps into a second line if the table is 50% of the screen. In cases like those, we've used the <code>display:none</code> CSS property for elements to hide all elements not directly related to pagination. See the <a href="https://github.com/rackerlabs/supportservice-ui/blob/master/app/styles/table.less#L40-L50">LESS of narrow tables used in Support Services</a> for an example of this. Note that this has not been implemented in EncoreUI yet.</li>
+            <li><span class="msg-info">ENCORE-SPECIFIC:</span> The pagination is visible in this example to show how mark-up will be implemented. Generally, the pagination directive does not display if there is only one page of data available. We use AngularJS to make sure this doesn't display.</li>
+            <li>It's a known issue that the footer wraps into a second line if the table is 50% of the screen.  Use <code>display:none</code> CSS property to hide all elements not directly related to pagination.  See the CSS example below for an example of this.  This has not been implemented in EncoreUI.</li>
```
Comment (CITguy) [inline demo/styleguide/tables.html]: Please remove "of this".

> See the CSS example below for an example ~~of this~~.

**Comment by 100stacks** [comment]
@CITguy got it :+1:

**Comment by 100stacks** [comment]
screen shot 2015-10-09 at 2 04 33 pm

**Comment by CITguy** [comment]
I'm wondering, do we need the CSS comment in the code example? I'd remove it to remove potential confusion.

**Comment by 100stacks** [comment]
@CITguy okay will do.  Also for the last bullet:
> It's a known issue that the footer wraps into a second line if the table is 50% of the screen. Use `display:none` CSS property to hide all elements not directly related to pagination. This has not been implemented in EncoreUI.

Is the last sentence still applicable?

**Comment by CITguy** [comment]
@100stacks, yes the last sentence is still applicable, because it states that the example is not part of the framework styles and must be implemented manually.

**Comment by 100stacks** [comment]
@CITguy, sounds good!  I will use this as the last bullet and drop the CSS example.

**Comment by 100stacks** [comment]
screen shot 2015-10-12 at 1 27 52 pm

**Comment by glynnis** [comment]
LGTM

**Comment by CITguy** [comment]
@100stacks, so take the screenshot you have in [this comment](https://github.com/rackerlabs/encore-ui/pull/1301#issuecomment-146963144) but remove the CSS comment in the example. Don't remove the whole example, though.

**Comment by 100stacks** [comment]
@CITguy, my mistake in thinking you wanted the entire CSS example removed.

**Comment by 100stacks** [comment]
screen shot 2015-10-12 at 4 31 39 pm

**Comment by CITguy** [comment]
LGTM

@glynnis still good?

**Pull request merged**